### ISSUE TITLE
Auto Rotation when pushed on navigation stack

### ISSUE THREAD

**Issue by wlgardner** [sberrevoets/SDCSegmentedViewController#18]
This is a great component. However, I'm having an issue when the interface rotates to and from portrait and or landscape. The child view/container doesn't seem to be rotating properly and half the screen is black.
The Controller is being pushed from a navigation Controller programmatically. 

Any ideas or suggestion would be helpful.

**Comment by sberrevoets** [comment]
Yes, this probably needs to be fixed by forwarding rotation methods to the active child view controller. I'm not sure when I'll have time to fix that, but if you submit a PR with the proposed changes in it, I'll be able to merge that in much sooner.

Thanks for reporting!
